packages/nimble-components/build/generate-scss/source/index.js

```diff
@@ -23,7 +23,8 @@ const tokensFileHeader = `// Nimble Components Tokens SCSS
 // Requires using a <nimble-theme-provider> in the page.
 // For more information see https://github.com/ni/nimble/tree/main/packages/nimble-components#theming
 
-@import 'tokens-internal';
+@use 'tokens-internal' as *;
+@forward 'tokens-internal';
```

Comment:
Because we actually expect clients to use variables like `$ni-nimble-internal-icon-color`, we have to forward them.